code-challenges.test.js

```diff
@@ -15,43 +17,115 @@
 
 
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
-
 // a) Create a test with an expect statement using the variable provided.
-
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+// var colors1 = ["purple", "blue", "green", "yellow", "pink"]
 // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+// var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
 // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
-
 // b) Create the function that makes the test pass.
 
-
+//get a shell test
+    // run it to make it fail
+// creat a function that will take in an array arrayIn()
+//function should remove first item ... maybe .shift 
+//after first item, array should be shuffled.
+// maybe a shuffle method? math random
+
+////xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
+describe("function", () => {
+    test("a function that takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+      expect(arrayIn(["purple", "blue", "green", "yellow", "pink"])).toContain("blue", "green", "yellow", "pink")
+    expect(arrayIn(["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"])).toContain("indigo", "periwinkle", "ochre", "aquamarine", "saffron")
+    })
+})
+// I had to look for the .toContain and it only works if you remove the [] from the array
+var colors1 = ["purple", "blue", "green", "yellow", "pink"]
+var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
+const arrayIn = (array) =>{
+    oneLess = []
+    for (let i = 0; i < array.length; i++) {
```

Comment:
I appreciate your love of for loops but do we need to iterate over everything in order to remove one element?